activerecord/test/cases/validations/numericality_validation_test.rb

```diff
@@ -102,4 +102,12 @@ def test_virtual_attribute_with_scale
 
     assert_not_predicate subject, :valid?
   end
+
+  def test_validates_numericality_of_ruby_class_with_aliased_attribute
```

Comment:
Based on the other tests in this file, it seems like this should be:

```suggestion
  def test_aliased_attribute
```